src/providers/TokensProvider.tsx

```diff
@@ -63,7 +62,7 @@ export default function TokensProvider({
       }
       if (gauges.gauges) {
         // add gauge tokens
-        ;(Object.values(gauges.gauges) as Gauge[]).forEach((gauge) => {
+        Object.values(gauges.gauges).forEach((gauge) => {
```

Comment:
was that type unneeded?

Reply:
I see this error when I try the same thing
<img width="449" alt="Screen Shot 2022-05-24 at 6 55 32 PM" src="https://user-images.githubusercontent.com/7607886/170162850-0f019924-34de-4317-a042-351879d6c71d.png">

Reply:
I think there's some strange configuration mismatch going on with Eslint because in my workspace it gives me different results.